### PR TITLE
Add TypeDefinitions for TimelineJS3

### PR DESCRIPTION
**Pull request by MikeMatusz**
Add a new type definition.
- [x] checked compilation succeeds with `--target es6` and `--noImplicitAny` options.
- [x] has correct [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)
- [x] has a [test file](http://definitelytyped.org/guides/contributing.html#tests) with the suffix of  `-tests.ts` or `-tests.tsx`.

### CODE REVIEW

**Comment by MikeMatusz** [comment]
No npm module yet (https://github.com/NUKnightLab/TimelineJS3/issues/308), so naming based on GitHub repository.

File: timelinejs3/timelinejs3.d.ts [hunk 1]
```diff
@@ -0,0 +1,351 @@
+// Type definitions for timelinejs3
```
Comment (mhegazy) [inline timelinejs3/timelinejs3.d.ts]: is there a version associated with this package?

Comment (MikeMatusz) [inline timelinejs3/timelinejs3.d.ts]: I'm using these definitions with 3.4.1, which is the latest version. Should I add that in the comments somewhere?

Comment (mhegazy) [inline timelinejs3/timelinejs3.d.ts]: yes please. see http://definitelytyped.org/guides/contributing.html#header

**Pull request merged**